### ISSUE TITLE
Crashes when illegal arguments are made

### ISSUE THREAD

**Issue by kirichoi** [SynBioDex/pySBOL#27]
Python crashes when improperly formed function calls are made. 
For example, 
```
d = sbol.Document()
d.read('file')
seq = d.sequences
seq.next()
```
Crashes Python. Need to implement fault checks.

**Comment by kirichoi** [comment]
@bbartley: Any ideas?

**Comment by hsauro** [comment]
This will be solved when a wrapper around SWIG.

**Comment by kirichoi** [comment]
Apply #34 treatment

**Comment by kirichoi** [comment]
Fixed by commit SynBioDex/libSBOL@1de702e5ebe47484c6918a24bafd5c5d213d9678
